### PR TITLE
Update test scaffold

### PR DESCRIPTION
**Pull request by mattpetrie**
@visormatt 
This add dependency imports for the new unit testing stack, and and updated test scaffold to the test file generator.

### CODE REVIEW

File: test/app.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ describe('generator-react-up TestComponent', function () {
 
   it('creates files', function () {
     assert.file([
-      '__test__/index.js',
+      '__test__/TestComponentSpec.js',
```
Comment (mattpetrie) [inline test/app.js:19]: It took a stab at updating the tests for the generator itself, but the yoeman test utilities seemed to be unable to deal with ES6 in template files. I looked into using babel but it was getting complicated, so ¯_(ツ)_/¯.

File: generators/app/index.js [hunk 2]
```diff
@@ -77,7 +77,7 @@ module.exports = yeoman.Base.extend({
 
     // Testing folder / stub
     if (props.addTests) {
-      var testPath = folder + '/__test__/index.js';
+      var testPath = folder + '/__test__/' + data.name + 'Spec.js';
```
Comment (visormatt) [inline generators/app/index.js:80]: This is up for debate but what I personally prefer `xxxxx.test.js` vs `xxxxxSpec.js` as I feel it's a bit more clear. I think this is one of those instances where we get team feedback on naming conventions. Won't stop me from merging this in, in the meantime.

File: test/app.js (hunk 1 above)
Comment (visormatt) [reply test/app.js:19]: Yeah, I have to figure out re-wiring the generator if we want to write it, in ES6.

**Review by visormatt — APPROVED** [review]
👀  good, I'll bump a patch and get this out. Thanks for updating it 👍

**Pull request merged**